internal/service/pg/resource_pg_test.go

```diff
@@ -598,3 +600,46 @@ resource "aiven_pg" "bar" {
 }
 `
 }
+
+func TestAccAivenPG_admin_creds(t *testing.T) {
```

Comment:
add a comment: we comment all functions, tests too

Reply:
[Not sure](https://github.com/aiven/terraform-provider-aiven/blob/1068fb7728f2c5df59c59e987ce5be330bff7f43/internal/service/pg/resource_pg_test.go#L15-L15) if that's so. OK I'll add comment to this.

Reply:
> [Not sure](https://github.com/aiven/terraform-provider-aiven/blob/1068fb7728f2c5df59c59e987ce5be330bff7f43/internal/service/pg/resource_pg_test.go#L15-L15) if that's so

we don't have necessary capacity to add comments everywhere, but we are supposed to be commenting each new function that we add